src/ansi.rs

```diff
@@ -1195,6 +1198,41 @@ impl StandardCharset {
     }
 }
 
+/// SCP control's first parameter which determines character path.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum ScpCharPath {
+    /// SCP's first parameter value of 0 (the default). Behavior is implementation defined.
+    ///
+    /// This value is defined by Terminals Working Group BiDi draft proposal.
```

Comment:
I don't really like this, considering `Terminals Working Group` is a dead project. A dead project's draft proposal isn't really the authority for anything and probably shouldn't be recommended in docs for developers new to the topic.

Behavior is best based on existing implementations and no reference to documentation might be better than fragile ones?